modules/services/autorandr.nix

```diff
@@ -0,0 +1,47 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.services.autorandr;
+
+in {
+  options = {
+    services.autorandr = {
+      enable = mkEnableOption "" // {
+        description = ''
+          Whether to enable the Autorandr systemd service.
+          This module is complementary to <code>programs.autorandr</code> which handles the
+          configuration (profiles).
+        '';
+      };
+
+      ignoreLid = mkOption {
+        default = false;
+        type = types.bool;
+        description =
+          "Treat outputs as connected even if their lids are closed";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.user.services.autorandr = {
```

Comment:
Seems this will only run once, when starting the graphical session? Shouldn't it somehow be hooked up to run after the screen setup changes?

Reply:
I agree, it would be even better. Do you know what would be the corresponding `systemd` target ?

Reply:
Sorry, I have no idea how to hook that up. I always use the builtin laptop screen.